File: JavaBasic/114_count_digits.java
// 114	114_count_digits.java	Count the number of digits in a given number.
public class _114_count_digits {
    public static void main(String[] args) {
        int num = 948321, count = 0, temp = num;
        while (temp > 0) { count++; temp /= 10; }
        System.out.println("Number: " + num + " | Digits: " + count);
    }
}

File: JSBASIC/114_count_digits.js
// 114	114_count_digits.js	Count the number of digits in a given number.

let num = 948321;
let count = 0;
let temp = num;
while (temp > 0) {
    count++;
    temp = Math.floor(temp / 10);
}
console.log("Number:", num);
console.log("Number of digits:", count);
